solidity-v1/dashboard/src/components/ProgressBar.jsx

```diff
@@ -26,18 +33,38 @@ const useProgressBarContext = () => {
   return context
 }
 
-const ProgressBar = ({ value, total, color, bgColor, children }) => {
+const ProgressBar = ({
+  value,
+  total,
+  color,
+  bgColor,
+  secondaryValue = null,
+  secondaryColor = null,
```

Comment:
This solution works OK only for 2 colors so I think we should refactor the `ProgresBarInline` component to be more flexible and we should be able to pass as many values as we want. Pseudo impl:

`ProgressBarInline.jsx`
```
<div className="progress-bar-wrapper" >
 {children}
</div>
```

`ProgressBarInlineItem.jsx`
```
<div
 className="progress-bar"
 style={{
 width: `${barWidth}%`,
  backgroundColor: color,
 }}
/>
```

And then in a parent component:
```
<ProgressBarInline>
 <ProgressBarInlinItem />
 <ProgressBarInlinItem />
 <ProgressBarInlinItem />
</ProgressBarInline>
```

Reply:
Great idea!

Fixed in 84df0563e90b72995d9c5b10d9751f8497d52f86